input/fsh/extensions/moped-ext-Altersgruppe.fsh

```diff
@@ -4,8 +4,29 @@ Id:           moped-ext-Altersgruppe
 Title:        "Altersgruppe" 
 Description:  "In Gruppen eingeteilt, wobei vollendete Lebensjahre ausschlaggebend sind."
 Context: MOPEDTransferEncounter
-* value[x] 0..1
-* value[x] only Coding
-* value[x] from Altersgruppe (required)
-* . ^definition =  "Nach dem Alter in Gruppen eingeteilt, wobei vollendete Lebensjahre ausschlaggebend sind."
 
+* extension contains
+    beiZugang 0..1 MS and
+    beiEntlassung 0..1 MS and
+    Neugeborenes 0..1 MS
+* extension[beiZugang].value[x] 0..1
+* extension[beiZugang].value[x] only code
+* extension[beiZugang].value[x] from Altersgruppe (required)
+* extension[Neugeborenes].value[x] 0..1
+* extension[Neugeborenes].value[x] only boolean
```

Comment:
ein vorschlag aus der AG Moped war, hier noch eine Invariant einzufügen, dass Neugeborenes nur True sein darf, wenn die Altersgruppe "0" ist. Falls das grad kompliziert ist, wär es gut das in ein separates Ticket auszulagern.